R/annotation.R

```diff
@@ -479,3 +479,200 @@ annotate_sv = function(sv_data,
   }
   return(all.annotated)
 }
+
+
+#' @title Annotate mutations target AID motif
+#'
+#' @description Checks for the presence of mutations at a given AID motif
+#'
+#' @details In positions that reference allele  "C" has been mutated, two nucleotides before and one nucleotide after it 
+#' will be captured. In positions that reference allele  "G" has been mutated, one nucleotide before and two nucleotides after it 
+#' will be captured. The other mutations will have "NO" for the sequence. 
+#' Then, the forward orientation and reverse compliment of the motif will be generated and their presence in aquired reference
+#' allele "C" and reference allele "G" sequence will be checked, respectively. ("NO" for the rows that have different reference
+#' alleles) 
+#'
+#' @param maf MAF data frame (required columns: Reference_Allele, Chromosome, Start_Position, End_Position)
+#' @param motif The motif sequence (default is WRCY)
+#' @param projection The genome build projection for the variants you are working with (default is grch37)
+#' @param fastaPath Can be a path to a FASTA file
+#'
+#' @return A data frame with two extra columns (seq and motif).
+#'
+#' @rawNamespace import(IRanges, except = c("merge", "shift", "collapse", "union", "slice", "intersect", "setdiff", "desc", "reduce")) 
+#' @rawNamespace import(GenomicRanges, except = c("merge", "shift", "union", "intersect", "setdiff", "reduce"))
+#' @import Rsamtools readr dplyr
+#' @export
+#'
+#' @examples
+#' findMotif(maf = maf,
+#'           motif = "WRCY",
+#'           projection = "hg38"
+#'          )
+
```

Comment:
Name suggestion. Either of these is more descriptive than the current function name

`annotate_mutation_motif_context`

`annotate_ssm_motif_context`

Reply:
Sure! I'm going to change it to `annotate_ssm_motif_context`. Thanks!

Reply:
Comments were addressed and the function now has the new name.